src/wallet-widget/example/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wallet Widget Shell</title>
+    <link rel="stylesheet" href="../style.css">
+</head>
+<body>
+    <script src="../widget.js"></script>
+    <script>
+        const config = {
+            logo_button_light: './assets/images/logo_vertical_light.png',
+            logo_header_light: './assets/images/logo_horizontal_light.png',
+            logo_button_dark: './assets/images/logo_vertical_dark.png',
+            logo_header_dark: './assets/images/logo_horizontal_dark.png',
+            theme: 'light'
```

Comment:
A structure like the one below does not expose the structure of the shell (button, header) and can be extended more easily if we add more themes or orientations.

```suggestion
            assets: {
              logos: {
                vertical: {
                  light: './assets/images/logo_vertical_light.png',
                  dark: './assets/images/logo_vertical_dark.png',
                },
                horizontal: {
                  light: './assets/images/logo_horizontal_light.png',
                  dark: './assets/images/logo_horizontal_dark.png',
                },
             },
            theme: 'light',
```